README.md

```diff
@@ -15,17 +15,16 @@ Make sure you have a [current version of Node.js](https://nodejs.org/en/about/re
 Getting Started
 ===============
 
-You can configure two main aspects of your new NEAR project:
+To create a new NEAR project with default settings, run:
 
-* frontend: do you want a **vanilla JavaScript** stack or **React**?
-* backend: do you want your contracts to be written in **Rust** or **AssemblyScript** (a dialect of TypeScript)?
+    npx create-near-app new-awesome-project
 
-| command                                                | frontend   | backend        |
-| ------------------------------------------------------ | ---------- | -------------- |
-| `npx create-near-app new-awesome-app`                  | React      | AssemblyScript |
-| `npx create-near-app --vanilla new-awesome-app`        | vanilla JS | AssemblyScript |
-| `npx create-near-app --rust new-awesome-app`           | React      | Rust           |
-| `npx create-near-app --vanilla --rust new-awesome-app` | vanilla JS | Rust           |
+This will use [npx](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner) to create a project with a **vanilla JavaScript** frontend and an [AssemblyScript](https://docs.near.org/docs/roles/developer/contracts/assemblyscript) smart contract.
```

Comment:
Should we consider changing to plain JavaScript? I think that's a more standard language, although maybe less fun. The benefit of being understandable regardless of cultural background is definitely high.

Reply:
I really like this direction, but if we change it here I wonder if we should also change it everywhere else. The help text when you type `create-near-app --help` mentions "vanilla"; the template name is `vanilla` – where do we change it to `plain` and where do we keep `vanilla`? The current approach at least has the virtue of consistency. Let's keep as-is for now and think about this more before we change it.